tests/include/Klocation_included

```diff
@@ -0,0 +1,6 @@
+
+comment "some comment"
+
+
+config MULTI_DEF
+    bool
```

Comment:
Thanks for adding some tests!

Comments:

 - There is already a `tests/sub/` directory. We could reuse that one instead of adding `tests/include/`.

 - Maybe `Klocation_included` could be renamed to `Klocation_sourced`. Then the new file could be called `Klocation_rsourced`.

 - I think `Klocation_rsourced` could have just `config MULTI_DEF` in it. A symbol defined in multiple locations doesn't need to specify the type at every location, and the comment seems to be unused.

 - Could you rename `EXPANDED_FROM_ENV` to `EXPANDED_IN_SOURCE` as well? Then `INCLUDE_DIR_FROM_ENV` could be called `EXPANDED_IN_RSOURCE`.

Thanks!

Reply:
Implemented per your description, with an exception that I've used 'TESTS_DIR_FROM_ENV' and 'SUB_DIR_FROM_ENV' names for envvar and config symbols.